Makefile

```
@@ -0,0 +1,72 @@
+.PHONY: install dev lint format check test clean play train train-tiny train-small train-full eval
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)